build/nsm/docker/Dockerfile

```diff
@@ -1,10 +1,15 @@
-FROM golang:1-alpine as build
+FROM golang:1 as build
 ENV PACKAGEPATH=github.com/ligato/networkservicemesh/
-RUN apk add --update protobuf git bash
+
+RUN apt update
+RUN apt install -y git bash unzip
+RUN wget https://github.com/google/protobuf/releases/download/v3.5.1/protoc-3.5.1-linux-x86_64.zip
```

Comment:
I like this implicit install better than whatever version apk was giving us above.